src/ios-deploy/ios-deploy.m

```diff
@@ -718,9 +719,9 @@ mach_error_t transfer_callback(CFDictionaryRef dict, int arg) {
 
     if (CFEqual(status, CFSTR("CopyingFile"))) {
         CFStringRef path = CFDictionaryGetValue(dict, CFSTR("Path"));
+        int overall_percent = percent / 2;
 
-        if ((last_path == NULL || !CFEqual(path, last_path)) && !CFStringHasSuffix(path, CFSTR(".ipa"))) {
-            int overall_percent = percent / 2;
```

Comment:
Since **`last_overall_percent`** really only pertains to this BundleCopy phase I think it would make more sense to make it a `static` variable rather than a global.

Reply:
Moved `last_path` as well to make it static.

Reply:
Thanks